Sources/SwerverCore/Router/Router.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+public class Router {
+    
+    private let responseBuilder = ResponseConstructor()
+    private let responseFormatter = ResponseFormatter()
+    
+    public init() {}
+    
+    public func process(request: HttpRequest) -> Data {
```

Comment:
This interface is a bit off. It accepts the parsed `HttpRequest`, but returns the formatted `HttpResponse` as `Data`. This is strange because they are two different levels of abstraction. I would expect the return type to be an `HttpRequest`.